contrib/win32/openssh/AppveyorHelper.psm1

```diff
@@ -289,6 +291,23 @@ function Publish-Artifact
 #>
 function Invoke-OpenSSHTests
 {
+    Set-BasicTestInfo -Confirm:$false
+    Invoke-OpenSSHSetupTest
+    if (($OpenSSHTestInfo -eq $null) -or (-not (Test-Path $OpenSSHTestInfo["SetupTestResultsFile"])))
+    {
+        Write-Warning "Test result file $OpenSSHTestInfo["SetupTestResultsFile"] not found after tests."
+        Write-BuildMessage -Message "Test result file $OpenSSHTestInfo["SetupTestResultsFile"] not found after tests." -Category Error
+        Set-BuildVariable TestPassed False
+    }
+    $xml = [xml](Get-Content $OpenSSHTestInfo["SetupTestResultsFile"] | out-string)
+    if ([int]$xml.'test-results'.failures -gt 0) 
+    {
+        $errorMessage = "$($xml.'test-results'.failures) setup tests in regress\pesterTests failed. Detail test log is at $($OpenSSHTestInfo["SetupTestResultsFile"])."
+        Write-Warning $errorMessage
+        Write-BuildMessage -Message $errorMessage -Category Error
+        Set-BuildVariable TestPassed False
```

Comment:
setup has failed, so do we stop executing further?

Reply:
currently we don't. but I can make the change to stop further tests if setup fail. similar for uninstall, only it when all previous tests passed. What do you think?